client/components/Tooltip/index.tsx

```diff
@@ -0,0 +1,45 @@
+// @flow
+import React from 'react';
+import Tippy from '@tippy.js/react';
+import 'tippy.js/dist/tippy.css';
+
+export interface TipProps {
+  content: string;
+  style?: Object;
+  children: React.ReactElement;
```

Comment:
Can we have option for different positions? And make sure the it is covered in Storybook.